source/comInterfaces/coreAudio/audioclient/__init__.py

```diff
@@ -0,0 +1,155 @@
+# This file is a part of pycaw library (https://github.com/AndreMiras/pycaw).

```

Comment:
Why don't we include pycaw as a dependency?

Reply:
Pycaw has its own dependencies that we don't need. I don't want to pollute NVDA with subdependency libraries that we don't need.

Reply:
As far as I can see, the only extra dependency Pycaw introduces is psutil. It is very likelike that py2exe will strip that, so in the end, there's no extra dependency.

Reply:
1. IIRC psutil has a bunch of dependencies of its own.
2. Didn't know about py2exe is that smart.
3. pycaw is missing one of the com interfaces - I had to add it myself. Sure it can be argued that we can import pycaw and write that new interface in a separate file - but current solution is more concise.
4. pycaw is just a wrapper around com interfaces. It is marginally more useful when working with raw com interfaces. But NVDA is working with raw com interfaces all over the place. Plus the new com interface will have to be interacted with outside of pycaw object model. So in my mind there is not much utility in pycaw object model.
5. Another PR that I'm planning - microphone mute - required changes to pycaw object model as well. So again it would be more convenient to work with raw com interfaces.

Reply:
I think generally we should avoid copying code from external dependencies where possible.
I think it would be much more elegant to use pycaw as a pip dedependency and patch as needed from NVDA, and/or make upstream PRs to fix pycaw.